projects/packages/changelogger/src/AddCommand.php

```diff
@@ -266,10 +266,15 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
 			// Determine the changelog entry and add to the file contents.
 			$entry = $input->getOption( 'entry' );
 			if ( $isInteractive ) {
+				if ( str_contains( getcwd(), '/projects/plugins/' ) ) {
```

Comment:
```suggestion
				if ( false !== strpos( getcwd(), '/projects/plugins/' ) ) {
```

I'm hesitant to require PHP 8 for the changelogger, forcing everyone to update. I think, generally, most of us are running whatever came with OS X for now (though, notably, MacOS 12.0 won't include PHP at all, so everyone will need to install their own version, but I digress).

Using this instead should work fine.

Reply:
Nice catch, thanks! TIL to watch out for PHP version requirements.